src/particles/beam/BeamParticleContainer.cpp

```diff
@@ -307,7 +272,49 @@ void BeamParticleContainer::TagByLevel (const int current_N_level,
 }
 
 void
-BeamParticleContainer::InSituComputeDiags (int islice, int islice_local)
+BeamParticleContainer::intializeSlice (int slice, int which_slice) {
+    HIPACE_PROFILE("BeamParticleContainer::intializeSlice()");
```

Comment:
Cleaning for later: Could you make CamelCase consistently? I don't want to suggest it here as it needs cleaning throughout the code then